README.md

```diff
@@ -4,4 +4,9 @@ Module 1:creating Hello Word Exercise
 .A .css file that includes one class with styles for your header (make it in your favorite color).
 .A test.md file with the text "This file should be ignored by git".
 .Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
-.Open a pull request
\ No newline at end of file
+.Open a pull request
+
+
```

Comment:
- [ ] **[OPTIONAL]** As a final note, you made changes to the `master` branch which should have been made to this very `hello-feature` branch. Generally, your main branch should only contain initial setup of the repo. All changes should only be made in a different feature branch.